src/main/resources/application.base.conf

```diff
@@ -3,6 +3,7 @@ flush = false
 core {
 
   threads = 3
+  healthcheckHash = "1234"
```

Comment:
please set it as healthCheckHash and add a minor description what it is in comments